docs/source/app-dev/services.rst

```diff
@@ -207,7 +207,7 @@ Utility Services
 Party Management Service
 ========================
 
-Use the **party management service** to allocate parties on the ledger and retrieve information about allocated parties.
+Use the **party management service** to allocate parties on the ledger, update participant-local party propertiesgi and retrieve information about allocated parties.
```

Comment:
```suggestion
Use the **party management service** to allocate parties on the ledger, update participant and local party properties and retrieve information about allocated parties.
```

Reply:
Pawel did mean "participant-local", but that's probably too much of a developer slang. Maybe `party properties local to participant`?

Reply:
"party properties local to the participant" would work I think.

Reply:
Thanks, changing to `update party properties local to the participant`